miasm2/os_dep/common.py

```diff
@@ -73,14 +73,13 @@ def vm_alloc(self, vm, size, perm=PAGE_READ | PAGE_WRITE):
         @perm: permission flags (PAGE_READ, PAGE_WRITE, PAGE_EXEC or any `|`
             combination of them); default is PAGE_READ|PAGE_WRITE
         """
-        addr = self.next_addr(size+8)
-        vm.add_memory_page(addr, perm, "\x00" * (size+8),
+        addr = self.next_addr(size)
+        vm.add_memory_page(addr, perm, "\x00" * (size),
```

Comment:
Tiny remark: you left your parenthesis around `(size)`, which lead to a diff on this function for the whole PR, where there is no need to